### ISSUE TITLE
Abysmal CPU performance ?

### ISSUE THREAD

**Issue by Panjaksli** [Ilya114/Box64Droid#131]
Hi, I've been comparing how well this emulator performs with chroot agains Winlator that is supposedly running under proot.
I've tested CB R23, and the scores are simply horrible.
This emulator achieves only 592 points vs 1440 under Winlator, whilst the CPU usage is at most 42%, even though all cores seemed to be utilized and boosting to max frequency in some form. I've tried enabling all fast box64/86 flags in config and updating from the menu, but that didn't hellp at all either.
Any idea where might be the issue ?
Android 14, Hyper OS
SD8Gen2, 12 GB RAM

**Comment by Panjaksli** [comment]
Yeah nevermind... After reboot the X11 app refuses to connect, lol.
Guess I'm using Winlator anyways

**Comment by Ilya114** [comment]
Benchmarks sometimes cant see real performance, you need to test in games
